### ISSUE TITLE
Add a way for Friendica leaders to announce changes to all Friendica instances

### ISSUE THREAD

**Issue by davidspring** [friendica/friendica#13119]
### Is the feature request related to a problem? Please describe.

Friendica leaders have agreed to make a major change to the meaning of the words Group, Forum and Circle. While this change will help new people moving from Facebook and Twitter understand how to use Friendica, it may create problems for existing users of Friendica. Apparently, Friendica does not have a way to inform all existing users of Friendica of changes other than posting the change to major Friendica forums. 

### Describe the feature you'd like

Joomla has a way through an Announcements section in their Admin panel to notify administrators of all Joomla websites when a major change has been made. Normally I am opposed to new features because they lead to more work for all concerned. However, I think being able to communicate with all Friendica administrators is so important that Friendica should add an Announcements section to the Friendica admin area to make Friendica administrators aware that a major change has been made. This is in addition to making announcements on the Friendica website and forums. 

### Describe alternatives you've considered

I do not think there is another option.  I guess the Friendica leaders can just sent out an email to all Friendica administrators. But I really think that an Announcements page needs to be added to the Admin panel. 

### Additional context
I am a retired college instructor who taught courses in CSS for more than 15 years. I  now write instructional manuals on how to use open source projects like Joomla and Linux Debian. I have been studying Friendica for about a year.  I plan on proposing a number of changes to the Friendica Frio theme **style dot CSS** file that I hope will greatly improve the appearance and usability of the default Frio theme on both mobile and desktop devices.  If these changes are approved, I think existing Friendica administrators may want to know about it as they will be able to dramatically improve the appearance of their sites in less than a couple of minutes (just by installing and renaming the style dot css file). . 

In addition, I think that Friendica is about to become much more popular as more people become aware of the censorship and data mining problems of Twitter and Facebook. This will in turn lead to more rapid changes in Friendica - just as Joomla has undergone dramatic changes during the past couple of years. These changes will require better communication between the leaders of Friendica and Friendica administrators.

**Comment by tobiasd** [comment]
What kind of message do you want to be delivered by this method, that cannot be delivered in the admin forum or with an announcement on the projects blog?

**Comment by nupplaphil** [comment]
I would think about things, Like the twitter add-on depreciations, which has an impact on all users.

**Comment by davidspring** [comment]
Yes. Could you please tell me about the current status of the twitter add on depreciation because I am having a very hard time getting current information about it!

**Comment by davidspring** [comment]
Also not everyone goes to the admin forum... what is the link to it?
Also not everyone goes to the project blog. Is there a way to subscribe to it? I apologize for my ignorance but I bet there are admins out there who are even dumber than I am.

**Comment by MrPetovan** [comment]
> Also not everyone goes to the admin forum... what is the link to it?

https://forum.friendi.ca/profile/admins

> Also not everyone goes to the project blog. Is there a way to subscribe to it?

RSS: https://friendi.ca/feed/

**Comment by loma-one** [comment]
Didn't we have a similar discussion when adding a Instanzblock generated a mail? At that time, it would be considered to inform users of an event of this kind via the notification system.

Question: would it make sense to extend the Notifyall addon so that Admin can choose to receive an email or a notification via the system notification?
It would then be the task of the admin of the instance to create the information and send it. In combination with the Page Header addon, the admin has two options at his disposal.

**Comment by bkil** [comment]
@MrPetovan This gave me an idea. Could we perhaps automatically subscribe the admin of a new instance to either the site RSS or to the News Forum and enable noisy notifications for posts coming in from there? It could be implemented in a polite way via an automatically ticked checkbox on one of the installation forms.

While at it, some other recommended forums (accounts to follow?) could also be included on the same form, as discovering these can take quite some time for most.

Related: #7795

**Comment by MrPetovan** [comment]
Why not but I'd rather it not being retroactive, which means we still wouldn't be able to reach most node admins this way.

**Comment by bkil** [comment]
After such a change is released, sending a friendly invite to all existing operators of all public instances is just a few clicks away.

Although, popping up such a recommended follow confirmation dialog (or an email to the same effect?) could also be implemented as an upgrade action that the admin account holder would see upon first login after they install the next release.
